plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -9,21 +7,135 @@
 		width: 100%;
 	}
 
+	.settings-payment-providers__header-title {
+		font-size: 16px;
+		font-weight: 600;
+		line-height: 24px;
+	}
+	.settings-payment-providers__header-select-container {
+		.components-input-control__prefix {
+			min-width: fit-content;
+			padding-left: 12px;
+			color: $gray-700;
+		}
+	}
 
-	table.wc_gateways {
-		.other-payment-methods__button-text {
-			margin-right: 4px;
+	// Override styles for woocommerce list items.
+	.woocommerce-list__item {
+		&:hover {
+			background-color: $white;
+
+			.woocommerce-list__item-title {
+				color: $gray-900;
+			}
 		}
+	}
+	.woocommerce-list__item-inner {
+		align-items: flex-start;
+		padding: 24px;
+	}
+
+	.woocommerce-list__item-title {
+		font-size: 14px;
+		color: $gray-900;
+		font-weight: 600;
+		display: inline-flex;
+		gap: 10px;
+		align-items: center;
+	}
+
+	.woocommerce-list__item-after {
+		&__actions {
+			display: flex;
+			gap: 16px;
+		}
+	}
 
-		td.other-payment-methods-row {
-			border-top: 1px solid #c3c4c7;
-			background-color: #fff;
+	.woocommerce-list__item-before img {
+		max-width: 96px;
+		height: 36px;
+		border-radius: 8px;
+	}
+
+	.other-payment-methods__button-text {
+		margin-right: 4px;
+	}
+
+	td.other-payment-methods-row {
+		border-top: 1px solid #c3c4c7;
+		background-color: #fff;
 
+	}
+
+	.other-payment-methods__image {
+		vertical-align: middle;
+		margin-right: 8px;
+	}
+	.other-payment-methods {
+		margin-top: 32px;
+
+		.components-panel__body.is-opened {
+			padding: $gap-large;
+			h2 {
+				margin: -24px -24px 5px;
+			}
+		}
+
+		button.components-panel__body-toggle {
+			font-size: 16px;
+			font-weight: 600;
+			line-height: 24px;
+			padding: $gap $gap-large !important;
+			&:focus {
+				box-shadow: none !important;
+			}
+			&:hover {
+				background: #fff !important;
+				box-shadow: none !important;
+			}
 		}
+	}
+	.other-payment-methods__grid {
+		display: grid;
+		grid-template-columns: repeat(3, 1fr); /* 3 columns */
+		gap: 16px; /* Space between columns */
```

Comment:
need to use `$gap` and similar

Reply:
will address in design issue https://github.com/woocommerce/woocommerce/issues/52113